lib/process-services-cloud/src/lib/services/web-socket.service.spec.ts

```diff
@@ -0,0 +1,135 @@
+/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AppConfigService, AuthenticationService } from '@alfresco/adf-core';
+import { TestBed } from '@angular/core/testing';
+import { Apollo, gql } from 'apollo-angular';
+import { of, Subject } from 'rxjs';
+import { WebSocketService } from './web-socket.service';
+import { SubscriptionOptions } from '@apollo/client/core';
+import { FeaturesServiceToken, IFeaturesService, provideMockFeatureFlags } from '@alfresco/adf-core/feature-flags';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+
+describe('WebSocketService', () => {
+    let service: WebSocketService;
+    let featureService: IFeaturesService;
+    const onLogoutSubject: Subject<void> = new Subject<void>();
+
+    const apolloMock = jasmine.createSpyObj('Apollo', ['use', 'createNamed']);
+
+    beforeEach(() => {
```

Comment:
There are some unit tests that were checking something useful that have been removed, maybe they need to be moved here?